src/PTFAnalysis.cpp

```diff
@@ -11,11 +11,13 @@
 #include <fstream>
 #include <math.h>
 
-// pulse charge (integrated pulse height over 2151ns to 2254ns)
-void PTFAnalysis::ChargeSum( float ped ){
+// pulse charge (integrated pulse height over time_low ns to time_high ns)
+// Optionally arguments: time_low and time_high (otherwise checks entire range)
+void PTFAnalysis::ChargeSum( float ped, int time_low, int time_high ){
+  if (time_high==0) time_high=hwaveform->GetNbinsX()*8;
```

Comment:
Really näive question again but what does time_low, time_high corresponds too ?

Reply:
Oops sorry, I forgot to change the corresponding variable names, the most recent commit and my other comment should clarify it better!